src/config.inc

```diff
@@ -1,10 +1,19 @@
 # Build platform (use one of AUTO, Unix, OSX_Universal, MSVC, Cygwin, MinGW)
 BUILD_ENV = AUTO
 
-# Variables you may want to override
-#CXXFLAGS = -Wall -O0 -g -Werror -Wno-long-long -Wno-sign-compare -Wno-parentheses -Wno-strict-aliasing -pedantic
+# Specify the C++ standard
 #CXXFLAGS = -std=c++11
```

Comment:
The rules in "common" take care of adding it as necessary. Setting `-std` to any other value will break compilation for "common" would add conflicting options.